cmd/exop/main.go

```diff
@@ -39,8 +39,9 @@ func main() {
 	store := statefile.New(statePath)
 
 	cfg := &server.Config{
-		VarDir: paths.VarDir,
-		Store:  store,
+		VarDir:     paths.VarDir,
+		Store:      store,
+		SyslogAddr: "localhost:4500", // XXX Configurable?
```

Comment:
Yeah, I didn't check if this port was available. I'm not quite sure what the right thing to do here is. I actually think maybe we should pick a port at random and write it in the run state / config file thing. What do you think?

Reply:
I think that is the right default behaviour, but it would still be nice to let users specify a specific port if they want to for whatever reason. This is another place where I think that it would be useful for `exo` to own the definition for the `LogProvider`. That way, there is one source of truth for the proxy to discover where to send logs.